scripts/validate-manifest.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+#
+# validate-manifest.sh [path/to/manifest.json]
+#
+# Dependencies: jq tr sort uniq
+#
+# Description: Ensures consistency of versions specified in manifest.json.
+
+set -eou pipefail
```

Comment:
Pipefail is a parameter to `-o`, if the previous worked I can imagine it might fail in some versions of bash.

```suggestion
set -euo pipefail
```